content/e20co/drone-marathon/index.md

```diff
@@ -0,0 +1,103 @@
+---
+author: "dellintosh"
+date: 2017-01-06T18:00:00-06:00
+image: "e20co/drone-marathon"
+repo: "e20co/drone-marathon"
```

Comment:
is this the github repository? I get a 404 when I visit github.com/e20co/drone-marathon